salt/utils/yamlloader.py

```diff
@@ -113,6 +117,10 @@ def construct_scalar(self, node):
                 node.value = eval(node.value, {}, {})  # pylint: disable=W0123
         return super(SaltYamlSafeLoader, self).construct_scalar(node)
 
+    def construct_yaml_str(self, node):
+        value = self.construct_scalar(node)
+        return salt.utils.locales.sdecode(value)
```

Comment:
Here are my thoughts. Happy to be corrected.

This needs to be utf-8 only.   For example someone edits a file on Unix in UTF-8 and then applies it to a windows system with a default language of Japanese.  Or the other way around someone create a file on Windows in Japanese and then puts in the salt master which is UTF-8, and applies it to Unix systems.

Salts python code is UTF-8, and therefore control/functions statements like pkg.installed are in UTF-8 used within sls files.

Compares some of the 0-255 8bit languages https://en.wikipedia.org/wiki/ISO/IEC_646 Take note of the following chars
`[`, `]`, `\` `{`, `}`, `^`, they are displayed as different characters in different languages.